src/app/pages/forms-and-validation/phone-number-format/phone-number-format.component.scss

```diff
@@ -0,0 +1,42 @@
+@import '~@pxblue/colors/palette.scss';
+
+section.mat-typography {
+    padding-top: 40px;
+    max-width: 480px;
+    margin: 0 auto;
+    .phone-number-form {
+        display: flex;
+        .country-code-select {
+            max-width: 112px;
+        }
+        .phone-number-input {
+            margin-left: 16px;
+            flex: 1;
+        }
+    }
+}
+
+::ng-deep .pxb-drawer-layout-nav-content {
+    background-color: map-get($pxb-white, 50);
+}
+
+::ng-deep .mat-form-field.mat-focused.mat-primary .mat-select-arrow {
+    border-top: 0;
+    border-bottom: 5px solid;
+}
+
+::ng-deep .cdk-overlay-container .cdk-overlay-pane .selectPanel {
+    min-width: 112px !important;
+    border-bottom-right-radius: 4px;
+    border-bottom-left-radius: 4px;
```

Comment:
What are these doing?

Reply:
These styles work with `!important`, but it would be nice to get them working without that.

Reply:
Removed it.